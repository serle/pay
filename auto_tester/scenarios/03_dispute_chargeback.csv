type,client,tx,amount
deposit,1,1,100.0
deposit,1,2,50.0
dispute,1,1,
chargeback,1,1,
