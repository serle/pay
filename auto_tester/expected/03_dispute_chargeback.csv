client,available,held,total,locked
1,50.0000,0.0000,50.0000,true
